frontend/src/components/navbar/Navbar.tsx

```diff
@@ -1,31 +1,67 @@
-import React, { useContext } from "react";
+import React, { useContext, useState } from "react";
 import { Link } from "react-router-dom";
 import { LoginContext } from "../login/LoginManager";
 import "./Navbar.scss";
+import Hamburger from "hamburger-react";
 
 export const Navbar: React.FunctionComponent = () => {
   const loginContext = useContext(LoginContext);
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  function toggleMenu() {
+    if (isExpanded) {
+      setIsExpanded(false);
+    }
+  }
 
   return (
     <nav className="navbar" role="navigation" aria-label="main navigation">
+      <div className="mobile-menu">
+        <Hamburger toggled={isExpanded} toggle={setIsExpanded} />
+      </div>
       <Link to="/">
         <img
           className="navbar__logo"
           src="/logo.png"
           alt="Whale Spotting logo"
         />
       </Link>
-      <div>
-        {!loginContext.isLoggedIn ? (
-          <div></div>
-        ) : (
-          <Link to="/">
-            <a className="button is-primary" onClick={loginContext.logOut}>
-              <strong>Logout</strong>
-            </a>
-          </Link>
-        )}
-      </div>
+      <ul className={`menu-items ${isExpanded ? "expanded" : ""}`}>
+        <Link to="/" onClick={() => toggleMenu()}>
```

Comment:
Ah, it makes sense why the `toggleMenu()` method only ever closes the menu - you only ever use it that way!

I think it would be better to explicitly close the menu in these cases - toggling the menu _works_, but you actually _always_ want to close the menu when one of its links is clicked. (You can actually remove the `toggleMenu()` method after changing this!)

```suggestion
        <Link to="/" onClick={() => setIsExpanded(false)}>
```